report/2.Analysis/LanguagesForYoungPeople.tex

```diff
@@ -0,0 +1,63 @@
+\section{Languages for beginners}
+
+There are a wide variety of different tools, games and languages with the desired intend of learning students how to program. 
```

Comment:
Line 3:
 * Who/what "desires" the intend? Remove "desired" from sentence.
 * Change "learning" to "teaching"